deployment/ccip/changeset/testhelpers/feestest/helpers.go

```diff
@@ -70,8 +70,8 @@ func RunFeeTokenTestCase(tc FeeTokenTestCase) {
 	expectedSeqNum := make(map[testhelpers.SourceDestPair]uint64)
 	expectedSeqNumExec := make(map[testhelpers.SourceDestPair][]uint64)
 
-	srcChain := tc.env.Chains[tc.src]
-	dstChain := tc.env.Chains[tc.dst]
+	srcChain := tc.env.BlockChains.EVMChains()[tc.src]
```

Comment:
same here, we create it once and use the result
```suggestion
        evmChains := tc.env.BlockChains.EVMChains
	srcChain := evmChains[tc.src]
```